debian/base/Dockerfile

```diff
@@ -20,10 +20,11 @@ LABEL io.confluent.docker.git.id=$COMMIT_ID
 ARG BUILD_NUMBER=-1
 LABEL io.confluent.docker.build.number=$BUILD_NUMBER
 
-ARG CONFLUENT_DEB_REPO="http://packages.confluent.io"
-ARG APT_ALLOW_UNAUTHENTICATED=false
+ARG CONFLUENT_PACKAGES_REPO=$CONFLUENT_PACKAGES_REPO
+
+ARG ALLOW_UNSIGNED=false
 #Set an env var so that it's available in derived images
-ENV APT_ALLOW_UNAUTHENTICATED=${APT_ALLOW_UNAUTHENTICATED}
+ENV ALLOW_UNSIGNED=$ALLOW_UNSIGNED
```

Comment:
I just don't know that we even have a policy wrt this repo yet, or how it's been treated for past releases. /cc @samjhecht